src/frontend/web_application/src/components/form/TextFieldGroup/style.scss

```diff
@@ -1,11 +1,24 @@
 @import '../../../styles/common';
 
 .m-text-field-group {
+  position: relative;
+
   &__label {
     display: block;
   }
 
   &__errors {
-    margin-top: map-get($co-form__spacing, 'medium');
+    position: relative;
+    margin-top: -(map-get($co-form__spacing, 'medium'));
   }
 }
+
+.m-field-errors {
```

Comment:
there is already a FieldErrors component with its styles.

this might introduce bugs in styles : 
it is loaded only if a form contains a TextFieldGroup component and will be applied to all other errors